static/app/components/events/eventTags/index.tsx

```diff
@@ -32,6 +33,9 @@ export function EventTags({event, organization, projectSlug, location}: Props) {
     ? event.tags?.filter(tag => tag.key !== 'device.class')
     : event.tags;
 
+  const shouldDisplayTagTree =
+    location.query.tagsTree || organization.features.includes('event-tags-tree-ui');
+
```

Comment:
Will probably pull this out into a util to use for hiding promoted tags as well, but for now it can live here as its only callsite.